src/main/java/command/DeleteCommand.java

```diff
@@ -0,0 +1,37 @@
+package command;
+
+import task.Task;
+import task.TaskList;
+import utilities.Parser;
+
+/**
+ * DeleteCommand class is used to delete a task from the task list.
+ */
+public class DeleteCommand extends Command {
+    private int index;
```

Comment:
I like the way you name your Commands and its execute method! We actually name them in the same way!